frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -153,9 +86,9 @@ export interface SurveyMetricsQueries {
 }
 
 export const surveyLogic = kea<surveyLogicType>([
-    path(['scenes', 'surveys', 'surveyLogic']),
     props({} as SurveyLogicProps),
     key(({ id }) => id),
+    path((key) => ['scenes', 'surveys', 'surveyLogic', key]),
```

Comment:
@liyiy mind testing once more if everything is working fine for survey results? I did, and it seemed fine to me.

I think you had to create all those setters because `key` was missing from the path, hence why selectors weren't updating, because they were all on the same logic